helix-core/src/main/java/org/apache/helix/util/StatusUpdateUtil.java

```diff
@@ -55,6 +56,10 @@
 public class StatusUpdateUtil {
   static Logger _logger = LoggerFactory.getLogger(StatusUpdateUtil.class);
 
+  public static final boolean ERROR_LOG_TO_ZK_ENABLED =
+      HelixUtil.getSystemPropertyAsBoolean(SystemPropertyKeys.STATEUPDATEUTIL_ERROR_LOG_ENABLED, "false");
```

Comment:
`Boolean.getBoolean(propertyKey, propertyDefaultValue)` is simple and good enough?